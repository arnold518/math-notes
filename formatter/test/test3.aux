\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{nav}{\headcommand {\slideentry {0}{0}{1}{1/1}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {1}{1}}}
\@writefile{toc}{\beamer@sectionintoc {1}{Introduction}{2}{0}{1}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {1}{1}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {1}{1}}}
\@writefile{nav}{\headcommand {\sectionentry {1}{Introduction}{2}{Introduction}{0}}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{1}{2/2}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {2}{2}}}
\newlabel{theorem:1.2}{{3}{3}{Introduction}{theorem:1.2}{}}
\@writefile{snm}{\beamer@slide {theorem:1.2}{3}}
\newlabel{concept:Concept: Probability}{{3}{3}{Introduction}{concept:Concept: Probability}{}}
\@writefile{snm}{\beamer@slide {concept:Concept: Probability}{3}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{2}{3/3}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {3}{3}}}
\newlabel{definition:Definition: Function}{{3}{4}{Introduction}{definition:Definition: Function}{}}
\@writefile{snm}{\beamer@slide {definition:Definition: Function}{3}}
\newlabel{example:Example: Fibonacci Sequence}{{3}{4}{Introduction}{example:Example: Fibonacci Sequence}{}}
\@writefile{snm}{\beamer@slide {example:Example: Fibonacci Sequence}{3}}
\newlabel{proof:Proof (Sketch)}{{3}{4}{Introduction}{proof:Proof (Sketch)}{}}
\@writefile{snm}{\beamer@slide {proof:Proof (Sketch)}{3}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{2}{3/4}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {3}{4}}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{3}{5/5}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {5}{5}}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{3}{5/6}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {5}{6}}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{4}{7/7}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {7}{7}}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{5}{8/8}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {8}{8}}}
\newlabel{concept:Concept: Probability}{{9}{9}{Introduction}{concept:Concept: Probability}{}}
\@writefile{snm}{\beamer@slide {concept:Concept: Probability}{9}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{6}{9/9}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {9}{9}}}
\newlabel{concept:Concept: Probability}{{9}{10}{Introduction}{concept:Concept: Probability}{}}
\@writefile{snm}{\beamer@slide {concept:Concept: Probability}{9}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{6}{9/10}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {9}{10}}}
\newlabel{concept:Concept: Probability}{{9}{11}{Introduction}{concept:Concept: Probability}{}}
\@writefile{snm}{\beamer@slide {concept:Concept: Probability}{9}}
\newlabel{concept:Concept: Probability}{{9}{11}{Introduction}{concept:Concept: Probability}{}}
\@writefile{snm}{\beamer@slide {concept:Concept: Probability}{9}}
\newlabel{concept:Concept: Probability}{{9}{11}{Introduction}{concept:Concept: Probability}{}}
\@writefile{snm}{\beamer@slide {concept:Concept: Probability}{9}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{6}{9/11}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {9}{11}}}
\newlabel{concept:Concept: Probability}{{9}{12}{Introduction}{concept:Concept: Probability}{}}
\@writefile{snm}{\beamer@slide {concept:Concept: Probability}{9}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{6}{9/12}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {9}{12}}}
\@writefile{toc}{\beamer@sectionintoc {2}{Inserting an Image}{13}{0}{2}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {2}{12}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {2}{12}}}
\@writefile{nav}{\headcommand {\sectionentry {2}{Inserting an Image}{13}{Inserting an Image}{0}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{1}{13/13}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {13}{13}}}
\newlabel{fig:image_example}{{1}{14}{Inserting an Image}{fig:image_example}{}}
\@writefile{snm}{\beamer@slide {fig:image_example}{14}}
\@writefile{nav}{\headcommand {\beamer@partpages {1}{14}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {13}{14}}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {13}{14}}}
\@writefile{nav}{\headcommand {\beamer@documentpages {14}}}
\@writefile{nav}{\headcommand {\gdef \inserttotalframenumber {13}}}
\gdef \@abspage@last{14}
