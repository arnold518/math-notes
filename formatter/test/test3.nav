\headcommand {\slideentry {0}{0}{1}{1/1}{}{0}}
\headcommand {\beamer@framepages {1}{1}}
\headcommand {\beamer@sectionpages {1}{1}}
\headcommand {\beamer@subsectionpages {1}{1}}
\headcommand {\sectionentry {1}{Introduction}{2}{Introduction}{0}}
\headcommand {\slideentry {1}{0}{1}{2/2}{}{0}}
\headcommand {\beamer@framepages {2}{2}}
\headcommand {\slideentry {1}{0}{2}{3/3}{}{0}}
\headcommand {\beamer@framepages {3}{3}}
\headcommand {\slideentry {1}{0}{2}{3/4}{}{0}}
\headcommand {\beamer@framepages {3}{4}}
\headcommand {\slideentry {1}{0}{3}{5/5}{}{0}}
\headcommand {\beamer@framepages {5}{5}}
\headcommand {\slideentry {1}{0}{3}{5/6}{}{0}}
\headcommand {\beamer@framepages {5}{6}}
\headcommand {\slideentry {1}{0}{4}{7/7}{}{0}}
\headcommand {\beamer@framepages {7}{7}}
\headcommand {\slideentry {1}{0}{5}{8/8}{}{0}}
\headcommand {\beamer@framepages {8}{8}}
\headcommand {\slideentry {1}{0}{6}{9/9}{}{0}}
\headcommand {\beamer@framepages {9}{9}}
\headcommand {\slideentry {1}{0}{6}{9/10}{}{0}}
\headcommand {\beamer@framepages {9}{10}}
\headcommand {\slideentry {1}{0}{6}{9/11}{}{0}}
\headcommand {\beamer@framepages {9}{11}}
\headcommand {\slideentry {1}{0}{6}{9/12}{}{0}}
\headcommand {\beamer@framepages {9}{12}}
\headcommand {\beamer@sectionpages {2}{12}}
\headcommand {\beamer@subsectionpages {2}{12}}
\headcommand {\sectionentry {2}{Inserting an Image}{13}{Inserting an Image}{0}}
\headcommand {\slideentry {2}{0}{1}{13/13}{}{0}}
\headcommand {\beamer@framepages {13}{13}}
\headcommand {\beamer@partpages {1}{14}}
\headcommand {\beamer@subsectionpages {13}{14}}
\headcommand {\beamer@sectionpages {13}{14}}
\headcommand {\beamer@documentpages {14}}
\headcommand {\gdef \inserttotalframenumber {13}}
