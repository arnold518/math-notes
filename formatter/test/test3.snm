\beamer@slide {theorem:1.2}{3}
\beamer@slide {concept:Concept: Probability}{3}
\beamer@slide {definition:Definition: Function}{3}
\beamer@slide {example:Example: Fibonacci Sequence}{3}
\beamer@slide {proof:Proof (Sketch)}{3}
\beamer@slide {concept:Concept: Probability}{9}
\beamer@slide {concept:Concept: Probability}{9}
\beamer@slide {concept:Concept: Probability}{9}
\beamer@slide {concept:Concept: Probability}{9}
\beamer@slide {concept:Concept: Probability}{9}
\beamer@slide {concept:Concept: Probability}{9}
\beamer@slide {fig:image_example}{14}
