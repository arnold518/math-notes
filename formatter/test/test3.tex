\documentclass{beamer}
\usetheme{Madrid}
\usepackage{amsmath, amssymb}
\usepackage{xcolor}

\setbeamertemplate{footline}
{%
  \leavevmode%
  \hbox{%
  \begin{beamercolorbox}[wd=.5\paperwidth,ht=2.25ex,dp=1ex,center]{author in head/foot}%
    \usebeamerfont{title in head/foot}\insertsection
  \end{beamercolorbox}%
  \begin{beamercolorbox}[wd=.5\paperwidth,ht=2.25ex,dp=1ex,leftskip=2ex,rightskip=2ex,sep=0pt]{date in head/foot}%
    \hfill%
    \usebeamerfont{date in head/foot}%
    \insertsubsection%
    \hfill%
    \usebeamercolor[fg]{page number in head/foot}%
    \usebeamerfont{page number in head/foot}%
    \usebeamertemplate{page number in head/foot}%
  \end{beamercolorbox}}%
  \vskip0pt%
}

\let\oldpart\part
\renewcommand{\part}[1]{
    \oldpart{#1}
    \begin{frame}
        \centering
        \Huge \textbf{#1}
    \end{frame}
}
\setbeamertemplate{section in toc}[ball unnumbered]
\setbeamertemplate{subsection in toc}[ball unnumbered]
\setbeamertemplate{subsubsection in toc}[ball unnumbered]

\AtBeginSection[]{
    \begin{frame}
        \vfill
        \centering
        \Huge \textbf{\insertsection} \\
        \vspace{1em}
        \begin{minipage}{0.7\linewidth}
            \centering
            \normalsize
            \tableofcontents[
                sectionstyle=show/hide,
                subsectionstyle=show/shaded/hide,
                subsubsectionstyle=show/shaded/shaded/hide
            ]
        \end{minipage}
        \vfill
    \end{frame}
}

\AtBeginSubsection[]{
    \begin{frame}
        \vfill
        \centering
        \LARGE \textbf{\insertsection} \\
        \vspace{1em}
        \Large \textbf{\insertsubsection} \\
        \vspace{1em}
        \begin{minipage}{0.7\linewidth}
            \centering
            \normalsize
            \tableofcontents[
                sectionstyle=show/hide,
                subsectionstyle=show/shaded/hide,
                subsubsectionstyle=show/shaded/shaded/hide
            ]
        \end{minipage}
        \vfill
    \end{frame}
}

\AtBeginSubsubsection[]{
    \begin{frame}
        \vfill
        \centering
        \LARGE \textbf{\insertsection} \\
        \vspace{1em}
        \Large \textbf{\insertsubsection} \\
        \vspace{1em}
        \large \textbf{\insertsubsubsection} \\
        \vspace{1em}
        \begin{minipage}{0.7\linewidth}
            \centering
            \normalsize
            \tableofcontents[
                sectionstyle=show/hide,
                subsectionstyle=show/shaded/hide,
                subsubsectionstyle=show/shaded/shaded/hide
            ]
        \end{minipage}
        \vfill
    \end{frame}
}

% Define custom colors
\definecolor{mytheorem}{RGB}{41, 128, 185} % Dark blue for the theorem title
\definecolor{myconcept}{RGB}{192, 57, 43}  % Dark red for concept title
\definecolor{mydefinition}{RGB}{39, 174, 96} % Dark green for definition title
\definecolor{myexample}{RGB}{211, 84, 0}   % Dark orange for example title
\definecolor{myproof}{RGB}{127, 140, 141}  % Gray for proof title

% Themed background colors
\definecolor{mytheorem_bg}{RGB}{214, 234, 248}   % Light blue body
\definecolor{myconcept_bg}{RGB}{242, 215, 213}   % Light red body
\definecolor{mydefinition_bg}{RGB}{235, 247, 238} % Light green body
\definecolor{myexample_bg}{RGB}{250, 229, 211}   % Light orange body
\definecolor{myproof_bg}{RGB}{236, 240, 241}     % Light gray body

\newenvironment{mytheoremblock}[2]{
    \phantomsection\label{theorem:#1} 
    \setbeamercolor{block title}{bg=mytheorem,fg=white}
    \setbeamercolor{block body}{bg=mytheorem_bg,fg=black}
    \begin{block}{Theorem #1: #2}
}
{\end{block}}

\newenvironment{myconceptblock}[2]{
    \phantomsection\label{concept:#1} 
    \setbeamercolor{block title}{bg=myconcept,fg=white}
    \setbeamercolor{block body}{bg=myconcept_bg,fg=black}
    \begin{block}{Concept #1: #2}
}
{\end{block}}

\newenvironment{myexampleblock}[2]{
    \phantomsection\label{example:#1} 
    \setbeamercolor{block title}{bg=myexample,fg=white}
    \setbeamercolor{block body}{bg=myexample_bg,fg=black}
    \begin{block}{Example #1: #2}
}
{\end{block}}

\newenvironment{mydefinitionblock}[2]{
    \phantomsection\label{definition:#1} 
    \setbeamercolor{block title}{bg=mydefinition,fg=white}
    \setbeamercolor{block body}{bg=mydefinition_bg,fg=black}
    \begin{block}{Definition #1: #2}
}
{\end{block}}

\newenvironment{myproofblock}[2]{
    \phantomsection\label{proof:#1} 
    \setbeamercolor{block title}{bg=myproof,fg=white}
    \setbeamercolor{block body}{bg=myproof_bg,fg=black}
    \begin{block}{Proof #1: #2}
}
{\end{block}}

\begin{document}

\title{Custom Blocks in Beamer}
\author{Your Name}
\date{\today}

\frame{\titlepage}

\section{Introduction}

\begin{frame}[allowframebreaks]{Custom Blocks Example}

    \begin{mytheoremblock}{1.2}{Pythagoras}
        In a right-angled triangle, the square of the hypotenuse is equal to the sum of the squares of the other two sides.
        \begin{enumerate}
            \item First item
            \item Second item
            \item Third item
            \item asdf
        \end{enumerate}
    \end{mytheoremblock}
    
    \begin{myconceptblock}{Concept: Probability}
        The probability of an event is a measure of the likelihood that the event will occur.
    \end{myconceptblock}
    
    \framebreak

    \begin{mydefinitionblock}{Definition: Function}
        A function is a relation that assigns exactly one output to each input.
    \end{mydefinitionblock}
    
    \begin{myexampleblock}{Example: Fibonacci Sequence}
        The sequence starts with 0 and 1, and each subsequent number is the sum of the previous two.
    \end{myexampleblock}
    
    \begin{myproofblock}{Proof (Sketch)}
        The proof follows by induction on \(n\).
    \end{myproofblock}
    
\end{frame}


\begin{frame}[allowframebreaks]{Unordered List Example}
    \begin{itemize}
        \item First item
        \item Second item
        \item Third item
        \item asdf
        \item asdf
        \item asdf
        \item asdf
        \item asdf
        \item asdf
        \item asdf
        \item asdf
        \item asdf
        \item asdf
        \item asdf
        \item asdf
        \item asdf
        \item asdf
        \item asdf
        \item asdf
        \item asdf
    \end{itemize}
\end{frame}

\begin{frame}{Ordered List Example}
    \begin{enumerate}
        \item First item
        \item Second item
        \item Third item
    \end{enumerate}
\end{frame}

\begin{frame}[fragile]{Code Block Example}
    \begin{verbatim}
    printf("Hello World");
    \end{verbatim}
\end{frame}

\begin{frame}[allowframebreaks]{Automatic Frame Break Example}
    \begin{myconceptblock}{Concept: Probability}
        The probability of an event is a measure of the likelihood that the event will occur.
        \begin{enumerate}
            \item First item
            \item Second item
            \item Third item
            \item Third item
            \item Third item
            \item Third item
            \item Third item
            \item Third item
            \item Third item
            \item Third item
        \end{enumerate}
    \end{myconceptblock}
    \begin{myconceptblock}{Concept: Probability}
        The probability of an event is a measure of the likelihood that the event will occur.
    \end{myconceptblock}
    \begin{myconceptblock}{Concept: Probability}
        The probability of an event is a measure of the likelihood that the event will occur.
    \end{myconceptblock}
    \begin{myconceptblock}{Concept: Probability}
        The probability of an event is a measure of the likelihood that the event will occur.
    \end{myconceptblock}
    \begin{myconceptblock}{Concept: Probability}
        The probability of an event is a measure of the likelihood that the event will occur.
    \end{myconceptblock}
    \begin{myconceptblock}{Concept: Probability}
        The probability of an event is a measure of the likelihood that the event will occur.
    \end{myconceptblock}
\end{frame}

\section{Inserting an Image}

\begin{figure}[h]
    \centering
    \includegraphics[width=0.7\textwidth]{example-image-a} % Replace with actual filename
    \caption{Example of an Image in LaTeX}
    \label{fig:image_example}
\end{figure}

\end{document}
