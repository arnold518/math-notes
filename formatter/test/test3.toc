\beamer@sectionintoc {1}{Introduction}{2}{0}{1}
\beamer@sectionintoc {2}{Inserting an Image}{13}{0}{2}
