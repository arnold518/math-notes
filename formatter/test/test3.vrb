\frametitle{Code Block Example}
\begin{verbatim}
    printf("Hello World");
    \end{verbatim}
