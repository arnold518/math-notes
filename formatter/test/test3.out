\BOOKMARK [2][]{Outline0.1}{\376\377\000I\000n\000t\000r\000o\000d\000u\000c\000t\000i\000o\000n}{}% 1
\BOOKMARK [2][]{Outline0.2}{\376\377\000I\000n\000s\000e\000r\000t\000i\000n\000g\000\040\000a\000n\000\040\000I\000m\000a\000g\000e}{}% 2
